### PR TITLE
Added 'dcos package list'.

### PR DESCRIPTION
**Pull request by armandgrillet**

### CODE REVIEW

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run windows integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run windows integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run windows integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run windows integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run windows integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run windows integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Review by rgo3 — APPROVED** [review]

**Review by bamarni — APPROVED** [review]

File: pkg/cosmos/client.go [hunk 1]
```diff
@@ -55,6 +55,56 @@ func (c *Client) PackageDescribe(name string, version string) (*Description, err
...
+				Name:               p.PackageInformation.PackageDefinition.Name,
+				PackagingVersion:   p.PackageInformation.PackageDefinition.PackagingVersion,
+				PostInstallNotes:   p.PackageInformation.PackageDefinition.PostInstallNotes,
+				PostUninstallNotes: p.PackageInformation.PackageDefinition.PostUninstallNotes,
+				PreInstallNotes:    p.PackageInformation.PackageDefinition.PreInstallNotes,
+				Scm:                p.PackageInformation.PackageDefinition.Scm,
+				Selected:           p.PackageInformation.PackageDefinition.Selected,
+				Tags:               p.PackageInformation.PackageDefinition.Tags,
+				Version:            p.PackageInformation.PackageDefinition.Version,
+				Website:            p.PackageInformation.PackageDefinition.Website,
```
Comment (bamarni) [inline pkg/cosmos/client.go:91]: Was there a specific reason for not using the type from client-go directly?

File: pkg/cosmos/client.go [hunk 2]
```diff
@@ -55,6 +55,56 @@ func (c *Client) PackageDescribe(name string, version string) (*Description, err
...
+		CosmosPackageListV1Request: optional.NewInterface(dcos.CosmosPackageListV1Request{})})
+	if err != nil {
+		return nil, err
+	}
+
+	var ps []Package
+	pAppNames := make(map[string]int)
+	index := 0
+	for _, p := range desc.Packages {
+		if val, ok := pAppNames[p.PackageInformation.PackageDefinition.Name]; ok {
```
Comment (janisz) [inline pkg/cosmos/client.go:70]: What if we have two packages installed with different path? E.g. jenkins instance per team?

**Pull request merged**